recipes/cmocka/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cmocka" %}
+{% set version = "1.1.5" %}
+{% set version_major_minor = ".".join(version.split('.')[:2]) %}
+{% set so_version = "0.7.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cmocka.org/files/{{ version_major_minor }}/{{ name }}-{{ version }}.tar.xz
+  sha256: f0ccd8242d55e2fd74b16ba518359151f6f8383ff8aef4976e48393f77bba8b6
+
+build:
+  number: 0
+  run_exports:
+    # No correlation between library name and
+    # so names
+    # https://gitlab.com/cmocka/cmocka/-/issues/43
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - make   # [unix]
+    - ninja  # [not unix]
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libcmocka${SHLIB_EXT}  # [unix]
+    - test -f $PREFIX/lib/libcmocka${SHLIB_EXT}.{{ so_version }}  # [linux]
+    - test -f $PREFIX/lib/libcmocka.{{ so_version }}${SHLIB_EXT}  # [osx]
+    - test -f $PREFIX/include/cmocka.h      # [unix]
+    - if not exist %LIBRARY_BIN%\\cmocka.dll exit 1   # [win]
+    - if not exist %LIBRARY_LIB%\\cmocka.lib exit 1   # [win]
+    - if not exist %LIBRARY_INC%\\cmocka.h exit 1     # [win]
+
+about:
+  home: https://cmocka.org/
+  license: Apache-2.0
+  license_family: Apache
+  license_file: COPYING
+  summary: An elegant unit testing framework for C with support for mock objects.
+
+  description: |
+    cmocka is ...
+
+    an elegant unit testing framework for C with support for mock objects. It
```

Comment:
maybe remove the .... and the newline :wink:

Reply:
That's taken from their website. But ok